### ISSUE TITLE
Upgraded to v0.41.4 via PPA, but app prompts to 'update' to v0.20.2

### ISSUE THREAD

**Issue by ghalfacree** [mooltipass/moolticute#433]
As per screenshot: I have upgraded, using the Mooltipass PPA, to v0.41.4 on Ubuntu 19.04 (and I see a shiny new HIBP integration which I'm itching to try out) - but the app tells me that v0.20.2 has been released and asks if I would like to download the 'update' now. Doing so successfully downgrades me to v0.20.2, and the next time I run `apt` it upgrades me to v0.41.4 and the cycle begins again.

![image](https://user-images.githubusercontent.com/1130041/55760998-c0825e00-5a55-11e9-863b-5fdd9e4ccd22.png)

```
$ sudo apt install moolticute
[sudo] password for blacklaw: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages will be upgraded:
  moolticute
1 to upgrade, 0 to newly install, 0 to remove and 6 not to upgrade.
Need to get 2,258 kB of archives.
After this operation, 10.1 MB disk space will be freed.
Get:1 http://ppa.launchpad.net/mooltipass/moolticute/ubuntu cosmic/main amd64 moolticute amd64 0.41.4~cosmic [2,258 kB]
Fetched 2,258 kB in 0s (4,520 kB/s)  
(Reading database ... 230014 files and directories currently installed.)
Preparing to unpack .../moolticute_0.41.4~cosmic_amd64.deb ...
Unpacking moolticute (0.41.4~cosmic) over (0.20.2) ...
Setting up moolticute (0.41.4~cosmic) ...
Processing triggers for mime-support (3.60ubuntu1) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for gnome-menus (3.32.0-1ubuntu1) ...
Processing triggers for desktop-file-utils (0.23-4ubuntu1) ...
```

**Comment by limpkin** [comment]
yup, we're working on that! sorry about this....

**Comment by ghalfacree** [comment]
No worries - now I know the upgrade isn't an upgrade, I can just click 'no' and go about my day!

**Comment by limpkin** [comment]
:)

we've just released 0.41.5 which shouldn't have this bug. for the ppas we know rely on the standard apt-get upgrade process and not display any notification.... but we'll change that soon...

**Comment by ghalfacree** [comment]
@limpkin I can confirm that 0.41.5 makes the "Check For Updates" button go away, and doesn't prompt to update automatically either.

![image](https://user-images.githubusercontent.com/1130041/55914008-f81c1200-5bdd-11e9-8f96-9ba40d4b1fe9.png)
